### ISSUE TITLE
Where is the source code?

### ISSUE THREAD

**Issue by fixcz** [macos-fuse-t/fuse-t#68]
Hello,

appreciate your work but I have a bit of an issue with this project. There is no source code in the repository itself.

I know there is an option to unpack the pkg file and look inside (I don't have the tools here now, I'm on Linux). But that's not the right approach. The source code should be in the repository, so the other users could contribute to the project as well.

Your approach just contradicts with basic GIT and also GitHub principes and instead of that, it uses GitHub platform as a marketing platform, nothing more.

Fix

**Comment by vanyasem** [comment]
Duplicate of https://github.com/macos-fuse-t/fuse-t/issues/1

> The project consists of two components: libfuse and fuse-t server. libfuse is LGPL licensed and can be downloaded from here: https://github.com/macos-fuse-t/libfuse. You can modify and build it as you wish, the build instructions are provided in the README file. The fuse-t server on the other hand is a proprietary component, it's written in go and doesn't link to or includes anything GPL related, all respective copyright owners are mentioned in License.txt file, so please stop your FUD or I'll ban you from posting here

_Originally posted by @macos-fuse-t in https://github.com/macos-fuse-t/fuse-t/issues/1#issuecomment-1420347389_

**Comment by fixcz** [comment]
I’m sorry, but your approach contradicts the basic principles of GitHub. I wouldn’t trust this software. I believe you could benefit more by making the code open. There are licenses that allow the community to contribute without you losing the right to sell the software.

Also, please stop being rude. If you look at the reactions to your comment, you’ll see that no one appreciates this approach.

fix

**Comment by vanyasem** [comment]
@fixcz yeah, it's a shame. but I'm afraid there's nothing we can do about it _(I am not affiliated with FUSE-T, just provided you with a quote from the author)_

macFUSE went closed source at some point: https://www.theregister.com/2019/12/16/fuse_macos_closed_source/
and FUSE-T was never open source to begin with

I assume with the introduction of FSKit by Apple (https://developer.apple.com/documentation/fskit/) the significance of proprietary FUSE implementations would be lessened
It might become feasible for software vendors to support FSKit directly, eliminating the need for FUSE implementations altogether

If you know any projects that currently rely on FUSE for macOS, consider making an issue proposing looking into supporting FSKit
